qiskit/optimization/converters/quadratic_program_to_qubo.py

```diff
@@ -115,6 +120,10 @@ def get_compatibility_msg(problem: QuadraticProgram) -> str:
             msg += 'Only linear equality constraints are supported.'
```

Comment:
This message does not have a trailing whitespace, that the others do, presumably so they can be concatenated and still read easily.